test-integration/helpers/exercise-selector.coffee

```diff
@@ -0,0 +1,43 @@
+_ = require 'underscore'
+{expect} = require 'chai'
+selenium = require 'selenium-webdriver'
+
+SelectReadingsList = require './select-readings-dialog'
+UnsavedDialog = require './unsaved-dialog'
+Calendar = require './calendar'
+{TestHelper} = require './test-element'
+
+COMMON_ELEMENTS =
+
+  selectSectionsContainer:
+    css: '.homework-plan-exercise-select-topics'
+
+  addExercisesContainer:
+    css: '.add-exercise-list'
+
+  closeButton:
+    css: '.footer-buttons [aria-role="close"]'
+
+
+EXERCISE_SELECTOR_ELEMENTS =
+  inactiveExerciseCard:
+    css: '.card.exercise.panel.panel-default'
+
+  reviewExercisesButton:
+    css: '.exercise-summary .btn.-review-exercises'
+
+
+class ExerciseSelector extends TestHelper
+  constructor: (test, testElementLocator) ->
+    testElementLocator ?= css: '.add-exercise-list'
+    super test, testElementLocator, EXERCISE_SELECTOR_ELEMENTS, defaultWaitTime: 3000
+
+  selectExercises: (numExercises) ->
```

Comment:
ah, yep :smile: . I wonder if it would be more useful to have `isExerciseSelectedByNumber/ById` and `toggleExerciseByNumber/ById` but I can add those to the master list and just keep them unimplemented.